center/src/main/java/com/microsoft/hydralab/center/controller/TestTaskController.java

```diff
@@ -107,6 +109,9 @@ public Result<Object> runTestTask(@CurrentSecurityContext SysUser requestor,
             return Result.ok(result);
         } catch (HydraLabRuntimeException e) {
             return Result.error(e.getCode(), e);
+        } catch (IllegalArgumentException e) {
```

Comment:
What specific scenario is under this catch clause?

Reply:
Will catch the exception throwed by Asserts